RingPublishingGDPR/src/main/java/com/ringpublishing/gdpr/internal/task/ConsentVerifyTask.java

```diff
@@ -37,14 +44,15 @@ public void run(Runnable finishCallback)
         {
             Log.w(TAG, "Fail verify consents. Consents are empty");
             requestsState.setVerifyState(VerifyState.FAILURE);
+            ringPublishingGDPRListener.onError(RingPublishingGDPRError.ERROR_EMPTY_CONSENTS);
```

Comment:
tu chodzi o to ze nie ma w user defaults? to moze coś w stylu: error local consents storage empty?